src/main/java/de/dhbw/karlsruhe/grammar/generation/GrammarGeneration.java

```diff
@@ -0,0 +1,115 @@
+package de.dhbw.karlsruhe.grammar.generation;
+
+import de.dhbw.karlsruhe.models.Grammar;
+import de.dhbw.karlsruhe.models.GrammarRule;
+
+import java.util.*;
+
+public class GrammarGeneration {
```

Comment:
Achtung: In der Branch von @whole01 und mir wurde hier eine Abstraktion eingefügt, um die Klassen für die Generierung etwas zu vereinheitlichen.

Reply:
Die print Statements noch entfernen.